package org.data;

public class Exception {

	public static void main(String[] args) {
		//ArithmeticException
	int a , b ,c ;
	a=1/1;
	b= 2/2;
	try {
		c=3/0;
	} catch (ArithmeticException e) {
		System.out.println("handled");
	}	
	//null pointer 
	String s =null;
	try {
		System.out.println(s.indexOf(1));
	} catch (NullPointerException e) {
    System.out.println("null handled");
	}
	
	String s1 = "Java";
	int indexOf = s.indexOf(4);
	System.out.println();

	
	
	
	}

	}


